src/main/java/games/strategy/triplea/delegate/MoveDelegate.java

```diff
@@ -208,6 +208,12 @@ public void end() {
     // while loading.
     if (GameStepPropertiesHelper.isResetUnitStateAtEnd(data)) {
       resetUnitStateAndDelegateState();
+    } else {
+
+      // Only air units can move during both CM and NCM in the same turn so moved units are set to no moves left
+      final List<Unit> alreadyMovedUnits =
```

Comment:
Nit: For readability, consider renaming this variable `alreadyMovedNonAirUnits`.

Reply:
Done.